### ISSUE TITLE
Cannot synchronize with filesystem: No such master key

### ISSUE THREAD

**Issue by piotrek-k** [laurent22/joplin#6978]
tl;dr: My existing notes cannot be synchronized from Filesystem sync target

Hello,

I'm setting up a new Joplin instance. I want to import my existing notes. Here is an error that I see after specifying path to my notes:

![image](https://user-images.githubusercontent.com/5506878/198882822-8e93f82c-d039-440a-b863-0a76b60f0e08.png)

> Completed: 30/10/2022 14:59 (148ms)
> Last error: Error: No such master key: fd68c0bcbfc14a0e80b677316e1bb8e1

There is also an error in console in Dev tools:

```
Logger.ts:219 14:59:45: Synchronizer: Error: No such master key: fd68c0bcbfc14a0e80b677316e1bb8e1
    at setMasterKeyHasBeenUsed (node_modules/@joplin/lib/services/synchronizer/syncInfoUtils.js:300)
    at Synchronizer.<anonymous> (node_modules/@joplin/lib/Synchronizer.js:434)
    at Generator.next (<anonymous>)
    at fulfilled (node_modules/@joplin/lib/Synchronizer.js:5)
```
My notes are not encrypted. Isn't term "master key" connected to encryption feature? Might it be that Joplin tries to decrypt my notes?

## Environment

Joplin version:  2.8.8 (prod, linux, flatpak)
Platform:  Linux desktop
OS specifics:  Ubuntu 22.04

## Steps to reproduce

1. Install Joplin
2. Set sync to Filesystem sync, provide path
3. Click "Synchronize"

## Logfile

[joplin_logs.txt](https://github.com/laurent22/joplin/files/9896023/joplin_logs.txt)

**Comment by laurent22** [comment]
They have never been encrypted?

**Comment by piotrek-k** [comment]
Might be that I've tried this option a few months ago, but since then I've worked on unencrypted notes. I can see right now that all my notes on filesystem are plain text

**Comment by wh201906** [comment]
This problem might be similar to this one
https://discourse.joplinapp.org/t/error-no-such-master-key-new-install-on-ipad/27308

**Comment by laurent22** [comment]
Yes, probably. You can delete the "activeMasterKeyId" block in info.json to solve the issue. I don't quite understand how there can be a key ID there when there's no master key, but I'll add a fix to better handle this anyway.

**Comment by wh201906** [comment]
Is it necessary to backup the sync target directory first then modify the info.json?

**Comment by laurent22** [comment]
No, it's relatively safe to modify info.json. Just to be sure, you might want to keep a copy of the original info.json

**Comment by piotrek-k** [comment]
Thank you, deleting "activeMasterKeyId" solved my problem.
